enhancements/console/customize-perspectives.md

```diff
@@ -0,0 +1,278 @@
+---
+title: customize-perspectives
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@jerolimov"
+  - "@cvogt"
```

Comment:
API approvers should be from the API review team, this would likely be myself or @deads2k